tooling/v8-snapshot/cache/dev-darwin/snapshot-meta.cache.json

```diff
@@ -0,0 +1,3453 @@
+{
+  "norewrite": [
+    "./evil-dns/evil-dns.js",
+    "./get-stream/buffer-stream.js",
+    "./graceful-fs/polyfills.js",
+    "./lockfile/lockfile.js",
+    "./node_modules/@babel/traverse/lib/path/comments.js",
+    "./node_modules/@babel/traverse/lib/path/conversion.js",
+    "./node_modules/@babel/traverse/lib/path/family.js",
+    "./node_modules/@babel/traverse/lib/path/introspection.js",
+    "./node_modules/@cypress/get-windows-proxy/src/registry.js",
+    "./node_modules/chalk/node_modules/supports-color/index.js",
+    "./node_modules/coffee-script/lib/coffee-script/helpers.js",
```

Comment:
have you looked to see if we can get ride of any of these dependencies?

Reply:
This definitely is helpful to look through for "what do we actually use". It'd be worth doing but probably separately.